src/Microsoft.Diagnostics.Monitoring.RestServer/Microsoft.Diagnostics.Monitoring.RestServer.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <NoWarn>;1591;1701</NoWarn>
     <Description>REST Api surface for dotnet-monitor</Description>
-    <IsPackable>false</IsPackable>
```

Comment:
Maybe add a comment here? We probably want to undo this once we have nuget packages we want to share